### ISSUE TITLE
Unable to launch "web", sqlite3.OperationalError

### ISSUE THREAD

**Issue by Tigzy** [cuckoosandbox/cuckoo#1579]
Hello, 
After configuration, I launch my 3 cuckoo services like this:

`cuckoo -cwd cuckoo` => OK
`cuckoo -cwd cuckoo api` => OK
`cuckoo --cwd cuckoo web -H 0` => NOK

I have the following error:
```
cuckoo@cuckoo-desktop:~$ cuckoo --cwd cuckoo web -H 0
Traceback (most recent call last):
  File "/usr/local/bin/cuckoo", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/main.py", line 488, in web
    Database().connect()
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/core/database.py", line 446, in connect
    self._create_tables()
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/core/database.py", line 454, in _create_tables
    "Unable to create or connect to database: %s" % e
cuckoo.common.exceptions.CuckooDatabaseError: Unable to create or connect to database: (sqlite3.OperationalError) unable to open database file
```
I have verified, in my cwd cuckoo there is that cuckoo.db file, and the whole directory has ownsership on cuckoo user

**Comment by jbremer** [comment]
Heh, confirmed. Give me a little bit.

**Comment by Tigzy** [comment]
So you reproduced? Thanks.

**Comment by jbremer** [comment]
Well, unfortunately I'm having some issues with properly resolving this (it's been on the todo list for a while). Going to have to postpone it once again ;-) Using absolute paths for the CWD should do the trick though.

**Comment by Tigzy** [comment]
Thanks, that's I wanted to know :) 
Using full path works.

**Comment by jbremer** [comment]
This was resolved in 898b49ce6e59484800b5d6806aa161eb5a17bb04 as part of #1424. You can already fetch it as part of `2.0.4a2`. Closing as such. Thanks!
